### PR TITLE
Syndistamp

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Adds a syndicate stamp.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Fox-McCloud** [comment]
Ahahah. Nice :+1:

File: code/modules/admin/verbs/striketeam.dm [hunk 1]
```diff
@@ -87,7 +87,10 @@ var/global/sent_strike_team = 0
...
 			P.name = "Spec. Ops Manual"
-			P.icon = "pamphlet-ds"
+			P.icon = "pamphlet-ds"		
+			var/obj/item/weapon/stamp/centcom/stamp = new
```
Comment (tigercat2000) [inline code/modules/admin/verbs/striketeam.dm]: conspiracy

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -468,6 +444,32 @@
 
 	add_fingerprint(user)
 	return
+	
+/obj/item/weapon/paper/proc/stamp(var/obj/item/weapon/stamp/S)
+	stamps += (!stamps || stamps == "" ? "<HR>" : "") + "<img src=large_[S.icon_state].png>"
+
+	var/image/stampoverlay = image('icons/obj/bureaucracy.dmi')
+	var/{x; y;}
```
Comment (tigercat2000) [inline code/modules/paperwork/paper.dm]: This is the most terrifying variable I have ever seen. It's relative-pathed variables on one line.

**Comment by KasparoVy** [comment]
Lmao hell yes :+1:

File: code/modules/paperwork/paper.dm (hunk 2 above)
Comment (TheDZD) [inline code/modules/paperwork/paper.dm]: This is absolutely horrifying.

Comment (Markolie) [inline code/modules/paperwork/paper.dm]: Don't hate, appreciate!

**Comment by SamCroswell** [comment]
Might add it to faxes sent by Syndicate.

**Comment by Markolie** [comment]
Yup, it's an option for admins who send faxes.

**Pull request merged**